tests/gis_tests/test_geoip2.py

```diff
@@ -75,80 +74,68 @@ def test02_bad_query(self):
             cntry_g.country_name(GeoIP2)
 
     @mock.patch('socket.gethostbyname')
-    def test03_country(self, gethostbyname):
+    def test_country(self, gethostbyname):
         "GeoIP country querying methods."
         gethostbyname.return_value = '128.249.1.1'
         g = GeoIP2(city='<foo>')
 
         for query in (self.fqdn, self.addr):
-            self.assertEqual(
-                'US',
-                g.country_code(query),
-                'Failed for func country_code and query %s' % query
-            )
-            self.assertEqual(
-                'United States',
-                g.country_name(query),
-                'Failed for func country_name and query %s' % query
-            )
-            self.assertEqual(
-                {'country_code': 'US', 'country_name': 'United States'},
-                g.country(query)
-            )
+            with self.subTest(query=query):
```

Comment:
Would rather add `subTest()` separately as it makes it difficult to see what the other changes are.